blenderproc/python/types/EntityUtility.py

```diff
@@ -224,9 +231,27 @@ def convert_to_entity_subclass(blender_object: bpy.types.Object) -> "Entity":
         return Entity(blender_object)
 
 
-def delete_multiple(entities: List[Union["Entity"]]):
+def delete_multiple(entities: List[Union["Entity"]], remove_all_offspring: bool = False):
     """ Deletes multiple entities at once
 
     :param entities: A list of entities that should be deleted
+    :param remove_all_offspring: If this is True all children and their children are recursively deleted
     """
-    bpy.ops.object.delete({"selected_objects": [e.blender_obj for e in entities]})
+    def collect_offspring(node: Entity) -> List[Entity]:
+        """
+        Recursively collects the offspring for a node
+        """
+        offspring = [node]
+        for child in node.get_children():
+            offspring.extend(collect_offspring(child))
+        return offspring
```

Comment:
Can we make this a method of `Entity`? I think this would be valuable also outside of this function

Reply:
Absolutely :D